### ISSUE TITLE
[Feature Request] player the message on load

### ISSUE THREAD

**Issue by bigenergy** [TerminatorNL/Tiquality#14]
If the player uses a high performance server, player ticking is slow. I suggest adding a message to the player if it creates a load. Since many players can not understand why their mechanisms slow down.
[sorry for my bad english :)) ]

**Comment by TerminatorNL** [comment]
Makes sense. I already thought about this problem a while back, and you can see that if you run `/tq profile`. It will tell you if your blocks run at full speed or not.

There's no distinction between very laggy or barely laggy, because we don't record all the update times per block. Instead I am subtracting the time it took to update from the granted number of seconds until it reaches zero, and then queue all future updates for later, but I don't know how long those queued updates will take.

If I were to implement this properly I have to find a way to do this automatically without causing too much overhead ram-wise and CPU wise, because the insane amounts of ticks I am hooked into.

Keep in mind that dead bushes etc also tick, and there's no way of knowing if they actually need to tick without overcomplicating the code.

A way to implement this would probably consist of periodically enabling the profiler on a tracker, but storing and analyzing those results costs tick time too (I have to get the block names in the server thread, etc).

In the end, however: LagGoggles will help players diagnose tick rates in Tiquality.

**TL;DR:**
I agree fully, and a future update of LagGoggles will take care of this later.

There's no ETA on this yet, as I want Tiquality 100% stable and reliable first.

**Comment by TerminatorNL** [comment]
This is actually already in Tiquality for a while now. fc71d4f1bae8a53acb29a836d358b0ced226cfdd -> [RelativeTPSTracker.java](https://github.com/TerminatorNL/Tiquality/blob/master/src/main/java/cf/terminator/tiquality/tracking/tickqueue/RelativeTPSTracker.java)
